libraries/AP_Landing/AP_Landing_Slope.cpp

```diff
@@ -351,7 +351,9 @@ int32_t AP_Landing::type_slope_get_target_airspeed_cm(void)
     case SlopeStage::APPROACH:
         if (land_airspeed >= 0) {
             target_airspeed_cm = land_airspeed * 100;
-        }
+        } else {
+            target_airspeed_cm = 0.5 * (aparm.airspeed_cruise_cm + aparm.airspeed_min * 100);
```

Comment:
this misses the one in mode_auto_target_airspeed_cm()

Reply:
we need a single function in AP_Landing for get_landing_airspeed() in m/s, then used in all places

Reply:
revised per above...